docs/quick-start/intellij-setup.md

```diff
@@ -1,45 +1,72 @@
 # IntelliJ setup
 > **_NOTE:_** The project now includes a .idea folder which should handle all set up for you.
-> This guide is incase that setup does not work as expected.
+> This guide is in case that setup does not work as expected.
 
-In this section, we will be going through the steps to setup the datapreparer tool to run in IntelliJ IDEA.
+This section will go through the steps to set up the datapreparer tool to run in IntelliJ IDEA.
 
-## Prerquesites
-Running the code in IntelliJ requires the following prequisites:
+## Prerequisites
+Running the code in IntelliJ requires the following prerequisites:
 
 - IntelliJ idea installed on your local machine
 - Java 11
 - Maven 3
-- The codebase cloned into a IntelliJ project and imported as a maven project with a Java 11 SDK configured.
+- The codebase cloned into an IntelliJ project and imported as a maven project with a Java 11 SDK configured.
 
-## Setting up the maven module
+## Setting the maven project
 Go to **File > Project structure** or <kbd>&#8984;</kbd> + <kbd>;</kbd> after doing this you should should see the following menu.
 ![Project structure](https://i.imgur.com/3JjeqR5.png){.center}
-At this point you should set the project SDK to Java 11 as well as the language level. 
-From here we go on to the modules tab, press the blue + button and choose the **new module** and the maven option.
-After completing this step you should get the following menu,
+At this point, you should set the project SDK to Java 11 and the language level. 
+We go on to the modules tab, press the blue + button and choose the **new module** and the maven option.
+After completing this step, you should get the following menu,
 ![New module](https://i.imgur.com/1Kr5Nkr.png){.center}
```

Comment:
the .centre shows up as text and the formatting is off, it should have some space between text and the image.